Builds/CMake/CMakeFuncs.cmake

```diff
@@ -6,7 +6,7 @@
 cmake_minimum_required(VERSION 3.1.0)
 
 if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
-  message(WARNING "In-source builds are strongly discouraged.")
+  message(WARNING "In-root builds are strongly discouraged.")
```

Comment:
Maybe I should change the message to remove the jargon entirely. `"Building in ${CMAKE_SOURCE_DIR} is strongly discouraged."`, and do the same in CMakeLists.txt.